L1Trigger/L1THGCalUtilities/python/hgcalTriggerChains.py

```diff
@@ -42,7 +48,28 @@ def register_chain(self, vfe, concentrator, backend1, backend2, selector='', ntu
         if ntuple!='' and not ntuple in self.ntuple: 
             raise KeyError('{} not registered as ntuplizer'.format(ntuple))
         self.chain.append( (vfe, concentrator, backend1, backend2, selector, ntuple) )
-    
+
+    def register_truth_chain(self, vfe, calo, selector='', ntuple=''):
```

Comment:
We might want to use a concentrator step, otherwise concentrator algos cannot be studied

Reply:
Hmm, maybe. Do you imagine that coming after the caloTruthProducer? I would just want to make sure we can also look at the "dummy" concentrator case, where we run directly on the truth cells. I suppose that is equivalent to the threshold 0 case, right?

Reply:
Actually I was thinking putting it before the "calo" producer, but I realized that it couldn't work for Super/CoarseTC. So let's just keep it that way for the moment.

Reply:
Ok, so just to be clear: Are you saying to put the concentrator in after the "calo" producer? Or just to leave it out for now. If we put it after, there's also the option of removing this chain and just adding the "calo" producer into the full TPG chain, with the default being some dummy trigger cell passthrough option. That might be a bit cleaner.

Reply:
Just leave it out for now. Unless you have a use-case for a chain with vfe+calo+concentrator